editoast/layers_description.yml

```diff
@@ -0,0 +1,223 @@
+layers:
+  - name: track_sections
+    table_name: osrd_infra_tracksectionlayer
+    id_field: id
+    views:
+      - name: geo
+        on_field: geographic
+        cache_duration: 3600
+        data_expr: track_section.data
```

Comment:
We can keep this like it was in chartos or we can have a static list of fields to integrate in the tile.

-------

We can do this later but the idea is to only integrate fields that are actually used by the front. Historically the fields in the tile were used for the editor (that's why they all had to be there). But this is not the case anymore.

Reply:
I have created an issue to do this later : https://github.com/DGEXSolutions/osrd/issues/2848